x-pack/plugins/siem/public/common/components/exceptions/viewer/index.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiPanel, EuiFlexGroup, EuiCommentList, EuiAccordion, EuiFlexItem } from '@elastic/eui';
+import React, { useEffect, useState, useMemo, useCallback } from 'react';
+import styled from 'styled-components';
+
+import { ExceptionDetails } from './exception_details';
+import { ExceptionEntries } from './exception_entries';
+import { getFormattedEntries, getFormattedComments } from '../helpers';
+import { FormattedEntry, ExceptionListItemSchema } from '../types';
+
+const MyFlexItem = styled(EuiFlexItem)`
+  &.comments--show {
+    padding: 16px;
+    border-top: 1px solid #d3dae6;
+  }
+`;
+
+interface ExceptionItemProps {
+  exceptionItem: ExceptionListItemSchema;
+  handleDelete: ({ id }: { id: string }) => void;
+  handleEdit: (item: ExceptionListItemSchema) => void;
+}
+
+const ExceptionItemComponent = ({
+  exceptionItem,
+  handleDelete,
+  handleEdit,
+}: ExceptionItemProps) => {
+  const [entryItems, setEntryItems] = useState<FormattedEntry[]>([]);
+  const [showComments, setShowComments] = useState<boolean>(false);
+
+  useEffect(() => {
+    const formattedEntries = getFormattedEntries(exceptionItem.entries ?? []);
```

Comment:
nit: I don't think this ?? does anything as the type states that it cannot be undefined or null?

<img width="1115" alt="Screen Shot 2020-06-03 at 10 52 26 AM" src="https://user-images.githubusercontent.com/1151048/83665219-5ea18380-a588-11ea-9d9e-7ea8e1a0ed80.png">

I would remove it if it doesn't do anything to keep the code clear with usage if it cannot be null/undefined.

Reply:
Yea, it's not doing anything there. Updated. Thanks!